common/api/serviceApi/service.api.types.ts

```diff
@@ -13,8 +13,8 @@ export interface IService {
   waterPriceTotal: number
   garbageCollectorPrice?: number
   inflicionPrice?: number
+  cleaning?: number
```

Comment:
IService should not have `cleaning` prop. Cleaning it's company related numeric value that represents how much company should pay for the cleaning according to the contract